pyERGM/metrics.py

```diff
@@ -727,6 +727,42 @@ def __str__(self):
         return "node_attribute_in"
 
 
+class SumDistancesConnectedNeurons(ExWeightNumEdges):
+    """
+    This function calculates the sum of euclidean distances between all pairs of connected neurons.
```

Comment:
really pedantic but it's a class, not a function

Reply:
True, changed (will be updated in the next commit)